iot_verifier/src/poc_report.rs

```diff
@@ -16,6 +16,7 @@ const REPORT_INSERT_SQL: &str = "insert into poc_report (
     report_type,
     status
 ) ";
+const REPORT_INSERT_CONFLICT_SQL: &str = " on conflict (id) do nothing ";
```

Comment:
if the poc reports table is the only thing db table we're interacting with in this module and since these consts aren't public it's probably okay, but having this independent query snippet without a table reference seems like it could be a footgun lying in wait for us; maybe ?
```suggestion
const REPORT_INSERT_CONFLICT_SQL: &str = " on conflict (poc_report.id) do nothing ";
```

Reply:
The alternative is to not use a const and just have a str on the query itself.  I think its better as a const, you would have to go to lengths to misuse this, no?

Reply:
I’m strictly advocating adding the context to the snippet in case this query ever gets extended in the future to join on any other tables, particularly since the compiler can’t check the use of any of these queries